### ISSUE TITLE
Yuzu crashes and closes when loading Streets Of Rage 4

### ISSUE THREAD

**Issue by AtlasSkyholder** [yuzu-emu/yuzu#8695]
I'm currently running Yuzu version 1118, but it has happened with previous versions.
When I load Streets of Rage 4, the loading bar appears, it attempts to load the game and then the FPS drops to 0 and crashes, then Yuzu closes.
I have tried using Vulkan and OpenGL, I have tried also two different versions of the game v1.0.3 and 1.0.7.
When I first started using Yuzu around february of 2022, I was experiencing the upside down problem, but a couple of months after the game just crashes and shuts down entirely.
I am running the latest drivers from AMD 22.7.1.
My guess is perhaps I'm running AMD hardware? I say this because at Yuzu compatibility/review page, people who commented on their experience with Streets of Rage 4 and how well it was running all had Nvidia GPUs.

Specs:
CPU:AMD 5600x
GPU: AMD RX 6700XT
Ram 32GB
Windows 10

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9230346/yuzu_log.txt)

**Comment by Tartifless** [comment]
I have exactly the same error, same info in log.

**Comment by AtlasSkyholder** [comment]
Follow up Update for the Yuzu team.

I just watched the most recent video from Mr. Sujano and he talked about the massive Yuzu update with version 1191, so I went ahead and updated it. My version is currently version 1192.

Streets of Rage 4 works now, it does not crash when trying to load it, the only problem is that the image is upside down, just like it was before for the Nvidia people. Also this is while using Vulkan. I played for a bit on the 1st chapter just to make sure the game would not crash, it was very stable, just upside down image.

Next, I tried it with OpenGL because I was just curious, well, OpenGL does load the game, it does not crash, but it has the same problem as Vulkan, it is upside down and worse, it is missing the background image, it only shows the start and the menu, no background image. I didn't even bother to try to go in game.

Anyways, it is almost there, the team at Yuzu rocks!!!!
